kolibri/plugins/coach/assets/src/state/actions/reports.js

```diff
@@ -234,24 +225,22 @@ function showReport(store, viewBy, params) {
   // API response handlers
   Promise.all(promises).then(
     ([report, contentSummary, userSummary]) => {
-      // save URL params to store
-      store.dispatch('SET_CLASS_ID', classId);
-      store.dispatch('SET_CHANNEL_ID', channelId);
-      store.dispatch('SET_CONTENT_SCOPE', contentScope);
-      store.dispatch('SET_CONTENT_SCOPE_ID', contentScopeId);
-      store.dispatch('SET_USER_SCOPE', userScope);
-      store.dispatch('SET_USER_SCOPE_ID', userScopeId);
-      store.dispatch('SET_ALL_OR_RECENT', ReportConstants.AllOrRecent.ALL);
-      store.dispatch('SET_VIEW_BY_CONTENT_OR_LEARNERS', viewBy);
-      store.dispatch('SET_SORT_COLUMN', sortColumn);
-      store.dispatch('SET_SORT_ORDER', sortOrder);
-
-      // save results of API request
-      store.dispatch('SET_TABLE_DATA', report || {});
-      store.dispatch('SET_CONTENT_SCOPE_SUMMARY', contentSummary);
-      store.dispatch('SET_USER_SCOPE_SUMMARY', userSummary || {});
-
-      // finih up
+      const pageState = {
+        classId,
+        channelId,
+        content_scope: contentScope,
```

Comment:
Is there a reason we're mixing `undersscore_case` with `camelCase` in the state? Or is this just to get things up and running?